### ISSUE TITLE
CalendarList not showing up until scroll

### ISSUE THREAD

**Issue by ipandz00** [wix/react-native-calendars#915]
Please make our job easier by filling this template out to completion. If you're requesting a feature instead of reporting a bug, please feel free to skip the Environment and Reproducible Demo sections.

## Description

CalendarList is not rendering until I try to scroll on the parent element.

## Expected Behavior

Everything should be rendered without me needing to scroll first.

## Observed Behavior

Container is empty, nothing is rendered.

## Environment

Please run these commands in the project folder and fill in their results:

* `npm ls react-native-calendars`: 1.212.0
* `npm ls react-native`: 0.59.5

Also specify:

1. Phone/emulator/simulator & version: iPhone Xr & iOS 12.2

## Reproducible Demo

Please provide a minimized reproducible demonstration of the problem you're reporting.

Issues that come with minimal repro's are resolved much more quickly than issues where a maintainer has to reproduce themselves.

**Comment by TeChn4K** [comment]
Same issue for me

**Comment by KantaP** [comment]
Same

**Comment by rakirox** [comment]
+1

**Comment by mustafaskyer** [comment]
add `removeClippedSubviews={false}` to CalendarList props solved it for me ✅
```js
<CalendarList
            removeClippedSubviews={false}
           ... props
/>
```

**Comment by emilisb** [comment]
Should be fixed by #1244
